src/vm/methodtable.cpp

```diff
@@ -2807,6 +2807,24 @@ bool MethodTable::ClassifyEightBytesWithNativeLayout(SystemVStructRegisterPassin
 
             continue;
         }
+        else if (cls == NFT_FIXEDBUFFER)
```

Comment:
This code path is same as NFT_NESTEDVALUECLASS, how about else if(cls == NFT_NESTEDVALUECLASS || cls == NFT_FIXEDBUFFER)